### ISSUE TITLE
Season is incorrect here in Australia

### ISSUE THREAD

**Issue by sebszocinski** [spencermountain/spacetime#352]
As of December 1st it's reporting winter when in fact it's summer. Weird thing is last week it was reporting spring which is correct but now it's gone to winter instead of summer.

**Comment by spencermountain** [comment]
thanks Seb, I'm gonna need some help to reproduce this - this works for me:
```js
let s = spacetime.now('australia/adelaide').minus(2, 'weeks')
s.every('day', s.add(4, 'weeks')).forEach(d => {
  console.log(d.format('iso-short'), d.season())
})
/*
2022-11-28 spring
2022-11-29 spring
2022-11-30 spring
2022-12-01 winter
2022-12-02 winter
2022-12-03 winter
2022-12-04 winter
2022-12-05 winter
*/
```
which timezone are you in?
thanks

**Comment by sebszocinski** [comment]
That's right. Your log is showing spring into winter. I'm based in Adelaide, Australia so that timezone is correct. It should be spring into summer as of Dec 1st.

**Comment by spencermountain** [comment]
Thank you! Will check this out in the next few days.
Cheers

**Comment by spencermountain** [comment]
oof - this has been a bug for about 7 years. Thank you for spotting it.
fixed in 7.3.0
